### PR TITLE
Accept multiple versions for consistent-versions rule

### PR DESCRIPTION
**Pull request by thasner**
@styu

### CODE REVIEW

File: packages/rules/src/consistentVersions.ts [hunk 1]
```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
     });
   }
 };
+
+const ensurePackageMatchesSomeVersion = (
```
Comment (thasner) [inline packages/rules/src/consistentVersions.ts:87]: I'm concerned that the `fixer` for multiple accepted values will be contentious, so to start off, I prefer adding this separate code path that doesn't attempt to fix when there are multiple `acceptedPackageDependencyValues`

File: packages/rules/src/consistentVersions.ts [hunk 2]
```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
...
+      }
+      return acceptedPackageDependencyVersion;
+    }
+  );
+
+  const actualPackageDependencyValue = packageJson.dependencies && packageJson.dependencies[dependencyPackageName];
+  const actualPackageDependencyVersion = coerce(actualPackageDependencyValue);
+  if (
+    actualPackageDependencyVersion != null &&
+    !acceptedPackageDependencyVersions.some(
```
Comment (styu) [inline packages/rules/src/consistentVersions.ts:111]: these suggestions are equivalent it's just how my brain wants to read this

```suggestion
    acceptedPackageDependencyVersions.every(
```

File: packages/rules/src/consistentVersions.ts [hunk 3]
```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
...
+      acceptedPackageDependencyVersion => actualPackageDependencyVersion.raw === acceptedPackageDependencyVersion.raw
```
Comment (styu) [inline packages/rules/src/consistentVersions.ts:112]:
```suggestion
      acceptedPackageDependencyVersion => actualPackageDependencyVersion.raw !== acceptedPackageDependencyVersion.raw
```

File: packages/rules/src/consistentVersions.ts [hunk 4]
```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
...
+    )
+  ) {
+    context.addError({
+      file: packageJsonPath,
+      message: `Expected dependency on ${dependencyPackageName} to match a version defined in monorepolint configuration '${JSON.stringify(
```
Comment (styu) [inline packages/rules/src/consistentVersions.ts:117]:
```suggestion
      message: `Expected dependency on ${dependencyPackageName} to one of '${JSON.stringify(
```

File: packages/rules/src/consistentVersions.ts [hunk 5]
```diff
@@ -79,3 +83,57 @@ const ensurePackageIsCorrectVersion = (
...
+  const actualPackageDevDependencyVersion = coerce(actualPackageDevDependencyValue);
+  if (
+    actualPackageDevDependencyVersion != null &&
+    !acceptedPackageDependencyVersions.some(
+      acceptedPackageDependencyVersion => actualPackageDevDependencyVersion.raw === acceptedPackageDependencyVersion.raw
+    )
+  ) {
+    context.addError({
+      file: packageJsonPath,
+      message: `Expected devDependency on ${dependencyPackageName} to match a version defined in monorepolint configuration '${JSON.stringify(
```
Comment (styu) [inline packages/rules/src/consistentVersions.ts:134]: same suggestions as above

File: packages/rules/src/consistentVersions.ts (hunk 2 above)
Comment (thasner) [reply packages/rules/src/consistentVersions.ts:111]: hmmmmm, my brain thought of "you must match one of these" as `if ("you don't match one of these") => Error`.

So I thought `if (!"you match one of these" ) => Error`,
as opposed to `if ("you don't match any of these") => Error`

The English for the latter seems smoother, though the code reads to me as `if ("all of these you do not match") => Error`.
This has been fun.
Gonna change it 👍

**Pull request merged**